### ISSUE TITLE
Move to crypto-browserify?

### ISSUE THREAD

**Issue by dcousens** [dcousens/hexxer#1]
Its not browserify related... but it is insanely useful.
Too many times have I been caught out by the incantations needed for `od` or `xxd` or `hexdump`.

Thoughts @fanatid?

**Comment by fanatid** [comment]
TBH, don't remember when I needed in such tool.

**Comment by fanatid** [comment]
OOI, why this written on C++ ? :)

**Comment by dcousens** [comment]
@fanatid cause I found it easier than writing it node haha.
I just needed it to be bullet proof.

**Comment by dcousens** [comment]
(recently had a few hours worth of time wasted due to bad hex parsing using one of the aforementioned tools - where no doubt I forgot the exact incantation)

**Comment by dcousens** [comment]
@fanatid nope? :)

**Comment by dcousens** [comment]
ping @fanatid

**Comment by dcousens** [comment]
@jprichardson?  thoughts?

**Comment by jprichardson** [comment]
Nah. I see the utility, but since it's not Browserify, not crypto, and not JavaScript, it doesn't feel like a good fit.
